### ISSUE TITLE
i18n, 404 section

### ISSUE THREAD

**Issue by skiff26** [vuejs/docs#2560]
## Description

When you try to go from a page on [vuejs.org](url) to [ua.vuejs.org](url) and [pt.vuejs.org](url), like this one - https://vuejs.org/guide/essentials/application.html, you'll run into an error. I only noticed this in the docs section. It only works if you didn't change pages before trying to switch to the UA/PT version.

![image](https://github.com/vuejs/docs/assets/109757944/87bcad3a-d2fb-475e-bb8a-682a3aca5a33)
![image](https://github.com/vuejs/docs/assets/109757944/5823ae49-1c3a-4c7e-aa80-60fc775cd5c5)

The same problem has [it.vuejs.org](url), an error section appears.

![image](https://github.com/vuejs/docs/assets/109757944/52f19f29-0100-40a0-8313-55c0d12c2e10)

**Comment by brc-dd** [comment]
I think those don't have an updated vitepress version or maybe some configuration issue at their netlify/vercel. It seems to be working fine for french, chinese, japanese, korean, bangla. Only ua, pt, it have issues IG.

**Comment by brc-dd** [comment]
seems to be a bug in theme - https://github.com/vuejs/theme/blob/main/src/vitepress/components/VPNavBarLocaleItems.vue#L18 (normalization should be done here and probably should use page.relativePath instead)

**Comment by skiff26** [comment]
So, do I need to recreate this issue in the vuejs/theme?

**Comment by brc-dd** [comment]
No need. I'll update both.

**Comment by skirtles-code** [comment]
I think #2664 is related.

**Comment by brc-dd** [comment]
Yeah, sorry. I actually forgot about this. I'll fix the theme. I don't think any of the translations sites are using `cleanUrls`, so that fix should work fine too 👀

**Comment by Jinjiang** [comment]
Same issue as https://github.com/vuejs-translations/docs-pt/issues/128
